src/main/Core/IniFileParser/IniFileParser.ts

```diff
@@ -26,19 +26,20 @@ export class IniFileParser implements IniFileParserInterface {
         for (const line of fileLines) {
             // New group
             if (line.startsWith("[") && line.endsWith("]") && !line.slice(1, -1).match(/\[\]/m)) {
-                if (result[line]) {
-                    throw `Ini Format Error! Duplicate groups found in ${fileString}`;
+                group = line.slice(1, -1);
+
+                if (result[group]) {
+                    throw new Error(`Ini Format Error. Duplicate group ${group} found`);
```

Comment:
Is this related to the application search extension? If not, please create a separate PR for it.

Reply:
Just a bug I caught while testing the app search extension. I'll make a separate PR for it